build/Utils.cmake

```diff
@@ -451,7 +451,6 @@ macro(optimize_default_compiler_settings)
 	  add_extra_compiler_option(-Werror=sequence-point)
 	  add_extra_compiler_option(-Wformat)
 	  add_extra_compiler_option(-Werror=format-security -Wformat)
-	  add_extra_compiler_option(-Wstrict-prototypes)
```

Comment:
pls explain

Reply:
In comment below you can see that compiler is producing a warning:
```
cc1plus: warning: command-line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++
```